[baidu.com] HTTPSConnectionPool(host='baidu.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbcd5c3efd0>, 'Connection to baidu.com timed out. (connect timeout=None)'))
[domaincontrol.com] HTTPSConnectionPool(host='domaincontrol.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcd593da90>: Failed to establish a new connection: [Errno 111] Connection refused'))
[macromedia.com] HTTPSConnectionPool(host='macromedia.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbcd593c7d0>, 'Connection to macromedia.com timed out. (connect timeout=None)'))
[msedge.net] HTTPSConnectionPool(host='msedge.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbcd5c3fd90>, 'Connection to msedge.net timed out. (connect timeout=None)'))
[qq.com] HTTPSConnectionPool(host='www.qq.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcd3b20910>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
[shifen.com] HTTPSConnectionPool(host='shifen.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbcd3b22fd0>, 'Connection to shifen.com timed out. (connect timeout=None)'))
[trbcdn.net] HTTPSConnectionPool(host='trbcdn.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'trbcdn.net'. (_ssl.c:1028)")))
